tests/ha/single_pod_restart/test_data_server_pod_restart_api.py

```diff
@@ -310,3 +321,182 @@ def test_chunk_upload_during_data_server_pod_restart(self):
                                       f"Actual checksum: {dnld_chksm}")
         LOGGER.info("Step 8: Successfully downloaded object and verified checksum")
         LOGGER.info("COMPLETED: Verify chunk upload during 1 data pod and 1 server pod restart")
+
+    @pytest.mark.ha
+    @pytest.mark.lc
+    @pytest.mark.tags("TEST-45532")
+    def test_copy_obj_during_data_server_pod_restart(self):
+        """
+        Verify copy object during 1 data pod and 1 server pod restart
+        """
+        LOGGER.info("STARTED: Verify copy object during 1 data pod and 1 server pod restart.")
+        bkt_cnt = HA_CFG["copy_obj_data"]["bkt_cnt"]
+        bkt_obj_dict = dict()
+        t_t = int(perf_counter_ns())
+        for cnt in range(bkt_cnt):
+            bkt_obj_dict[f"ha-bkt{cnt}-{t_t}"] = f"ha-obj{cnt}-{t_t}"
+        self.extra_files.append(self.multipart_obj_path)
```

Comment:
append only when you are sure that file is created.. may be on line 349.

Reply:
is this taken care? don't see the chnage though